circle.yml

```diff
@@ -20,3 +20,7 @@ test:
     override:
         - cd $GS_WD && go test -v $(glide novendor)
         - cd $GS_WD && go build -a -v -tags netgo -ldflags "-w -X main.gitCommit=$(git rev-parse --short HEAD)"
+        - cd $GS_WD &&
+          cp bindata.go bindata-repo.go &&
+          make update-bindata &&
+          cmp bindata.go bindata-repo.go
```

Comment:
How does the workflow look like then? Do I have to manually regenerate the bindata every time I change the templates? This would make me super sad. I want this to be automated.